client/jetpack-onboarding/steps/site-title.jsx

```diff
@@ -79,9 +76,24 @@ class JetpackOnboardingSiteTitleStep extends React.PureComponent {
 					title="Site Title ‹ Jetpack Start"
 				/>
 
-				<FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />
+				<FormattedHeader headerText={ headerText } />
 
 				<Card className="steps__form">
+					<img
+						className="steps__illustration"
+						src={ '/calypso/images/illustrations/illustration-start.svg' }
+						alt=""
```

Comment:
That's cheating your way around a11y errors 😛 

I think we should make it a habit to actually use these for some (at least somewhat) helpful information.
Something like "Person looking at stylized website" maybe?

Reply:
Hmm, reading https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img

> Setting this attribute to an empty string (`alt=""`) indicates that this image is not a key part of the content, and that non-visual browsers may omit it from rendering.

it's probably arguable that it's more decorational rather than a key part of the content, so maybe fine to leave as `alt=""` after all.

Reply:
I went through the same investigation and reached the same conclusion while adding the empty `alt` 🤣  Thanks for noticing anyway.

Reply:
Maybe it'd make sense to leave a comment next to that line tho... something like `// Illustration -- not a key part of the content`?

Reply:
We have quite a few of those cases around Calypso and I don't see such a comment anywhere. I prefer passing 🤐